### ISSUE TITLE
npm installation problems

### ISSUE THREAD

**Issue by modernity76** [HashLips/hashlips_art_engine#1526]
C:\NFT\hashlips_art_engine-1.1.2_patch_v6>node index.js
(node:1004) UnhandledPromiseRejectionWarning: Error: layer name can not contain dashes, please fix: black Skull T-shirt#10.png
    at C:\NFT\hashlips_art_engine-1.1.2_patch_v6\src\main.js:77:15
    at Array.map (<anonymous>)
    at getElements (C:\NFT\hashlips_art_engine-1.1.2_patch_v6\src\main.js:75:6)
    at C:\NFT\hashlips_art_engine-1.1.2_patch_v6\src\main.js:92:15
    at Array.map (<anonymous>)
    at layersSetup (C:\NFT\hashlips_art_engine-1.1.2_patch_v6\src\main.js:90:30)
    at startCreating (C:\NFT\hashlips_art_engine-1.1.2_patch_v6\src\main.js:356:20)
    at C:\NFT\hashlips_art_engine-1.1.2_patch_v6\index.js:6:3
    at Object.<anonymous> (C:\NFT\hashlips_art_engine-1.1.2_patch_v6\index.js:7:3)
    at Module._compile (internal/modules/cjs/loader.js:1114:14)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1004) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:1004) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by bolshoytoster** [comment]
@modernity76 did you read the first line?
```js
Error: layer name can not contain dashes, please fix: black Skull T-shirt#10.png
```

**Comment by modernity76** [comment]
I was a fool and solved the problem. Thank you.
